Documentation/data-collection.md

```diff
@@ -245,6 +245,9 @@ data:
     # (@openshift/storage, OpenShift Storage team) Persistent Volume usage metrics: this is the number of volumes per plugin
     # and per volume type (filesystem/block)
     - '{__name__="cluster:kube_persistentvolume_plugin_type_counts:sum"}'
+    # (Advanced Cluster Management) visual_web_terminal_sessions_total is the count of Visual Web Terminal sessions created 
+    # on the hub cluster.
+    - '{__name__="visual_web_terminal_sessions_total"}'
```

Comment:
What is the cardinality for this in total for this metric, I could not find the metric `visual_web_terminal_sessions_total` via search, do you mind sharing the repo where its created?

cc @smarterclayton I was told they have your 👍 already

Reply:
This metric has no labels so it will be a singleton per hub OCP cluster where ACM is installed.  So from a telemetry standpoint, there will be one occurrence of this counter for every ACM hub cluster participating in telemetry.

The metric is created in this repo here https://github.com/open-cluster-management/kui-web-terminal/blob/master/proxy/app/lib/metricsUtils.js